backend/src/models/referral.ts

```diff
@@ -0,0 +1,33 @@
+import { InferSchemaType, Schema, model } from "mongoose";
+
+const referralSchema = new Schema(
+  {
+    status: {
+      type: String,
+      required: true,
+      enum: ["Referred", "Viewing", "Pending", "Approved", "Denied", "Leased", "Canceled"],
+      default: "Referred",
+    },
+    renterCandidate: { type: Schema.Types.ObjectId, ref: "Renter" },
+    unitId: {
+      type: Schema.Types.ObjectId,
+      required: true,
+    },
+    // Will be set later on in the flow
+    assignedHousingLocatorId: {
+      type: Schema.Types.ObjectId,
+      required: false,
+    },
+    // Should be set to the user that created the referral (RS or HL)
+    assignedReferringStaffId: {
```

Comment:
```suggestion
    assignedHousingLocator: {
      type: Schema.Types.ObjectId,
      required: false,
    },
    // Should be set to the user that created the referral (RS or HL)
    assignedReferringStaff: {
```

As per previous suggestion